modules/ROOT/pages/addition/instance-requirements.adoc

```diff
@@ -17,6 +17,23 @@ This setting acts as an lower bound for any completed query to appear in the log
 Thus, if this value is set too high, the queries you want to monitor might not get logged. 
 Setting this value to zero will log every query.
 
+[[server_log_config]]
+=== Finding out the location of server log configuration (server-logs.xml)
+
+The location of server log configuration file is configured in `neo4j.conf` (see documentation on link:https://neo4j.com/docs/operations-manual/current/configuration/configuration-settings/#config_server.logs.config[server.logs.config] configuration setting).
+
+The currently effective location of server log configuration can be found out using the following Cypher query:
+
+[source, terminal, role=noheader]
+----
+./cypher-shell "CALL dbms.listConfig() YIELD name, value WHERE name='server.logs.config' RETURN value"
+----
+
+[IMPORTANT]
+====
+Check that the server log configuration exists, otherwise NOM agent will not be able to add query log collection configuration to it.
```

Comment:
```suggestion
Make sure that the server log configuration exists, otherwise NOM agent will not be able to add query log collection configuration to it.
```